angel.py

```diff
@@ -171,27 +174,20 @@ def write_to_csv(index, name01, *player_lists):
             cur_time = time.strftime("%Y-%m-%d %H-%M-%S")
             with open(f"{index} - {name01} - {cur_time}.csv", 'w', newline='') as f: # In Python 3, if do not put newline='' AND choose 'w' instead of 'wb', you will have an empty 2nd row in output .csv file.
                 writer = csv.writer(f, delimiter=',')
-                header = ['Telegram Username','Name','GenderPref','Gender','Interests','2truths1lie','Intro','House','CG','Year','Faculty'] # add header to output csv file
+
+                config_file = open("config.yml", 'r')
+                index_dict = yaml.full_load(config_file)["player_attribute_index"]
+                index_lst = {k: v for k, v in sorted(index_dict.items(), key=lambda item: item[1])}
+
+                header = [] # add header to output csv file
```

Comment:
Use linting tool to guide with comment formatting. In this case, it complains about needing two spaces before comment